sdk/anomalydetector/Azure.AI.AnomalyDetector/src/Generated/AnomalyDetectorClient.cs

```diff
@@ -67,6 +67,7 @@ public AnomalyDetectorClient(Uri endpoint, AzureKeyCredential credential, Anomal
             _apiVersion = options.Version;
         }
 
+        // The convenience method of this operation is not generated because the convenience method is not turned on in typespec file.
```

Comment:
Not sure if we need this one since this is an explicit choice by the customer.  We only want to flag when a customer is expecting both, but only gets a single one.

Reply:
OK - let me remove this message